addons/web/static/src/webclient/burger_menu/burger_menu.js

```diff
@@ -63,6 +63,7 @@ export class BurgerMenu extends Component {
     }
     _onMenuClicked(menu) {
         this.menuRepo.selectMenu(menu);
```

Comment:
This needs an await in front of it with the new implementation, to avoid flickering.

Reply:
Using wait does not close the burger menu. So we have to call _closeBurger method. discussed with @SimonGenin 

Thanks